### ISSUE TITLE
Black Screen Error

### ISSUE THREAD

**Issue by Salil03** [3b1b/manim#119]
The code compiles perfectly alright but when I open the WarpSquare.mp4 file it shows a black screen.

Why does this happen

**Comment by LJitJamesL** [comment]
I am having a similar issue. I had just finished installing Manim and its dependencies today, and the code runs fine barring the path not found error, but the videos all come out as nothing but a few seconds of a pitch black screen.

**Comment by Salil03** [comment]
Exactly LJitJamesL, I had the exact same issue

**Comment by Salil03** [comment]
Can anyone please help

**Comment by hmuar** [comment]
Had same issue. Haven't figured out solution but couple things I noticed that might help you investigate.

- When I use **--save_pngs** option, the images it generates were also black. So at least for me, I already had an issue outside of ffmpeg.
- I noticed the pixel data was properly being generated, so there was prop something going on with pushing that into a PIL Image, conversion, etc. When I tried commenting out the 
`image = image.convert(mode)` line in the `save_image` function in `scene.py`, the images generated using the **--save_pngs** option seemed to work. The diff is writing with RGBA mode vs trying to convert to RGB first. This doesn't fix the black video issue, but it at least confirmed the pixel data is properly being generated and it's all there.
- I was using the docker image for all of this ^ . So before I dug into figuring out why video was working, I tried just installing all of the dependencies directly myself, without docker. This got rid of the black video prob.
